internal/providers/pluginfw/resources/qualitymonitor/tests/resource_quality_monitor_test.go

```diff
@@ -48,7 +48,7 @@ resource "databricks_sql_table" "myInferenceTable" {
 
 `
 
-func TestUcAccQualityMonitorPluginFramework(t *testing.T) {
+func TestUcAccQualityMonitor(t *testing.T) {
```

Comment:
Since these are under pluginfw package, we can remove PluginFramework from the test name